src/components/Dashboard/SafeApps/Grid.tsx

```diff
@@ -0,0 +1,91 @@
+import { ReactElement, useMemo } from 'react'
+import styled from 'styled-components'
+import { Button } from '@gnosis.pm/safe-react-components'
+import { generatePath, Link } from 'react-router-dom'
+
+import { useAppList } from 'src/routes/safe/components/Apps/hooks/appList/useAppList'
+import { GENERIC_APPS_ROUTE } from 'src/routes/routes'
+import Card from 'src/components/Dashboard/SafeApps/Card'
+import ExploreIcon from 'src/assets/icons/explore.svg'
+
+const StyledGrid = styled.div`
+  display: flex;
+  align-items: center;
+  gap: 20px;
+`
+
+const StyledExplorerButton = styled.div`
+  width: 260px;
+  height: 200px;
+  background-color: white;
+  border-radius: 8px;
+  padding: 24px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  gap: 8px;
+`
+
+const StyledLink = styled(Link)`
+  text-decoration: none;
+
+  > button {
+    width: 200px;
+  }
+`
+
+// Transactions Builder && Wallet connect
+const officialAppIds = ['29', '11']
+const MAX_APPS = 3
+
+const Grid = (): ReactElement => {
+  const { allApps, pinnedSafeApps, togglePin } = useAppList()
+
+  const pinnedSafeAppsIds = useMemo(() => pinnedSafeApps.map((app) => app.id), [pinnedSafeApps])
+  const officialApps = useMemo(
+    () => allApps.filter((app) => officialAppIds.includes(app.id) && !pinnedSafeAppsIds.includes(app.id)),
+    [allApps, pinnedSafeAppsIds],
+  )
+  const displayedApps = useMemo(
+    () => pinnedSafeApps.concat(officialApps).slice(0, MAX_APPS),
+    [pinnedSafeAppsIds, officialApps],
```

Comment:
Wrong dependency. Please also rename `pinnedSafeAppsIds` to `pinnedSafeAppIds` (w/o an `s` after `App`).

Reply:
Oops, good catch. Fixed it.